src/caller.jl

```diff
@@ -358,49 +358,57 @@ function prepare_argument(x::Any)
     :ptr in fieldnames(typeof(x)) || error("unrecognized argument $x")
     if x.ptr isa libSingular.number_ptr
         ptr = x.ptr
-        rng = parent(x)
-        new_ptr = libSingular.n_Copy(ptr, rng.ptr)
+        ring_ptr = parent(x).ptr
+        new_ptr = libSingular.n_Copy(ptr, ring_ptr)
         return Any[mapping_types_reversed[:NUMBER_CMD], new_ptr.cpp_object], nothing
     end
     error("unrecognized argument $x")
 end
 
-function low_level_caller_rng(lib::String, name::String, ring, args...)
+function low_level_caller_rng(lib::String, name::String, ring, args)
     libSingular.load_library(lib)
-    arguments = Vector{Any}()
+    return low_level_caller_rng(name, ring, args)
+end
+
+function low_level_caller_rng(name::String, ring, args)
```

Comment:
```suggestion
function low_level_caller_ring(name::String, ring, args)
```
See my comment above.